src/Silex/Provider/SecurityServiceProvider.php

```diff
@@ -344,14 +371,14 @@ public function register(Container $app)
                 if (is_string($rule[0])) {
                     $rule[0] = new RequestMatcher($rule[0]);
                 } elseif (is_array($rule[0])) {
-                    $rule[0] += [
+                    $rule[0] += array(
```

Comment:
@stof My bad, as I presumed standard would be short array syntax as we're supporting versions only above 5.4.
@K0nias disregard my comment then :)